src/app/components/crypto-assets/choose-crypto-asset/crypto-asset-list-item.tsx

```diff
@@ -0,0 +1,31 @@
+import type { AllTransferableCryptoAssetBalances } from '@shared/models/crypto-asset-balance.model';
+
+import { CryptoCurrencyAssetItem } from '@app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item';
+
+import { CryptoCurrencyAssetIcon } from './crypto-currency-asset-icon';
+import { FungibleTokenAssetItem } from './fungible-token-asset-item';
+
+interface CryptoAssetListItemProps {
+  assetBalance: AllTransferableCryptoAssetBalances;
+  onClick: () => void;
```

Comment:
```suggestion
  onClick(): void;
```

this makes no material difference in the type, but would be nice to enforce a consistent way of defining fn types. Is there an eslint rule for this?

Reply:
there is! https://typescript-eslint.io/rules/method-signature-style/#method

Maybe we can add this rule to prefer methods here?

Reply:
Just found it too! :D yea will add it now